cleanup_pks.sh

```diff
@@ -2,5 +2,12 @@
 
 source `dirname "${BASH_SOURCE[0]}"`/util.sh
 
-# TODO uncomment if we use a PKS cluster per job
-# pks delete-cluster ${TS_G_ENV}-${CLUSTER_NAME} --non-interactive --wait
+TS_G_ENV=$(echo $TOOLSMITH_ENV | base64 --decode | jq -r .name)
+
+gcp_region=`gcloud config get-value compute/region`
+gcp_zone=`gcloud config get-value compute/zone`
+
+gcloud compute forwarding-rules delete ${TS_G_ENV}-${CLUSTER_NAME}-fw --region=${gcp_region}
+gcloud compute target-instances delete ${TS_G_ENV}-${CLUSTER_NAME}-ti --zone=${gcp_zone}
+gcloud compute addresses delete ${TS_G_ENV}-${CLUSTER_NAME}-ip --region=${gcp_region}
+travis_wait 60 pks delete-cluster ${TS_G_ENV}-${CLUSTER_NAME} --non-interactive --wait
```

Comment:
nit: deleting a cluster shouldn't need this, shouldn't hurt though other then eating stdout